www/style/codecheckerviewer.css

```diff
@@ -16,17 +16,30 @@ html, body {
 }
 
 #headerPane {
-  height: 33px;
+  height: 42px;
   background-color: #007ea7;
 }
 
-#logo {
+#logo-container {
   font-family: 'Raleway', sans-serif;
   font-size: 20pt;
   color: white;
 }
 
-#version {
+#logo {
+  background: left center no-repeat url(/images/logo.png);
```

Comment:
It may be written without quotes, or surrounded by single or double quotes.
https://developer.mozilla.org/en-US/docs/Web/CSS/url